versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -24,6 +24,32 @@ project, you must also update the
 composer require front-commerce/magento2-commerce-module@2.1.0
 ```
 
+### Adobe Commerce Negotiable quotes
+
+In this version, we added support for
+[Negotiable quotes](https://experienceleague.adobe.com/docs/commerce-admin/b2b/quotes/quotes.html).
+When updating your project, please note that some changes have been made on the
+following files:
+
+- `theme/components/atoms/Icon/icons/core-icons.js`
+- `theme/components/atoms/Icon/Icon.js`
+- `theme/modules/Cart/CartFooter.js`
+- `theme/modules/User/AccountNavigation/AccountNavigation.js`
+- `theme/layouts/Header/Navigation/AccountNavigation/AccountSubNavigation.js`
+- `theme/pages/Account/AccountLayout.js`
+- `theme/pages/Account/EnhanceAccount.js`
+
+If you have overridden the `src/web/theme/_b2b.scss` stylesheet, then you should
```

Comment:
actually this file has nothing special compared to the list above, so it should probably be integrated into it.